types/sd-notify/index.d.ts

```diff
@@ -0,0 +1,52 @@
+// Type definitions for sd-notify 2.8
+// Project: https://github.com/systemd/node-sd-notify#readme
+// Definitions by: Jan Beseda <https://github.com/Cheprer>
+//                 Rory Bradford <https://github.com/roryrjb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.7
```

Comment:
```suggestion
```
sorry, what's the language feature requiring this specific version?

Reply:
Nothing specific, it is what the project I'm intended to use it uses.

Reply:
So there is no requirement for TS version, we promote interop between types (like not enforcing specific TS version), can this one be removed?